app/main.py

```diff
@@ -1,8 +1,58 @@
 class Car:
-    # write your code here
-    pass
+    __slots__ = ("comfort_class", "clean_mark", "brand")
```

Comment:
an interesting implementation, I didn't know about __slots__ untill now, had to look for the info about it :)